astropy/io/fits/hdu/hdulist.py

```diff
@@ -965,10 +965,10 @@ def info(self, output=None):
             name = self._file.name
 
         results = ['Filename: {}'.format(name),
-                   'No.    Name         Type      Cards   Dimensions   Format']
+                   'No.    Name      Ver    Type      Cards   Dimensions   Format']
 
-        format = '{:3d}  {:10}  {:11}  {:5d}   {}   {}   {}'
-        default = ('', '', 0, (), '', '')
+        format = '{:3d}  {:10}  {:3} {:11}  {:5d}   {}   {}   {}'
```

Comment:
Yes, this seems reasonable. If someone has a FITS file with version number more than 3 digits, the person has a bigger problem than misaligned `fits.info()` print-out. 😉